### PR TITLE
Adding docs around JIRA issue triage

### PR DESCRIPTION
**Pull request by nikhilkh**

### CODE REVIEW

File: docs/jira-triage.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#JIRA Issue Triage Process
+
+##Goal
+Every issue needs to be triaged. A triaged issue is actionable and there is an eventual desire to resolve it and has all the fields set accurately. In particular, low quality issues and eliminated from the system. Also, critical issues need to be fixed asap and this process should highlight those.
```
Comment (TimBarham) [inline docs/jira-triage.md]: Typo - 'low quality issues ~~and~~*are* eliminated'

**Comment by nikhilkh** [comment]
@shazron @infil00p Please review.

**Comment by shazron** [comment]
Thanks Nikhil for writing the doc.

I'm still not too sure what the difference between Blocker and Critical is, and I'm sure most ppl triaging will have that confusion as well.

As Joe suggested, perhaps Blocker is either of these:
- mobile-spec doesn't build
- breaks app-hello-world (doesn't load, glitches)
- if it builds, it crashes mobile-spec

Thus, Blocker is Critical plus any of the three above criteria.
Critical is as you wrote: "This will cause the main function of the component to fail and needs to be fixed asap."

We used to have Component Owners, but that fell to the wayside because component owners were not doing the triaging, and others didn't want to pick up items that were already assigned. 

I think we should revive the Component Owner setting, with the caveat that if the PMC feels that there is no work done there, we re-assign, or make it clear anyone can work on any issue at any time (I prefer the latter). 

If you are working on an issue, you should assign it to yourself and set it to "In Progress" to flag that no one else should touch it (assignment does not signify you are working on it).

**Comment by riknoll** [comment]
Does it make sense to reference the platform owner as well as the component owner if there is trouble reproducing? Also, do we need to develop a criteria for what improvements/new features should remain open? A lot of them ask for features very specific to the reporter's use case.

**Comment by nikhilkh** [comment]
I made some more tweaks based on feedback. Please take another look. Thanks! What should we do with auto-assignment behavior?
https://issues.apache.org/jira/plugins/servlet/project-config/CB/components
I believe it primarily affects @purplecabbage  for Windows & WP8 and @TimBarham for Browser & CordovaServe.

**Comment by nikhilkh** [comment]
@purplecabbage - your thoughts on this and removing auto-assignment?

I am hoping to merge these guidelines today.

**Comment by purplecabbage** [comment]
I am fine with whatever.
The only reservations I have are simply that as an open source project we generally can't expect anyone to work on anything other than what they want to work on, or what they personally see as most important at the time.
One week I may have 30 hours to work on Cordova, the next week just 2, so it is hard to give people firm 'job/roles' within the project without a lot of overlap. That said, I am fine with being the go-to auto assign for all windows/wp8 tickets.  The important part in my mind is that everyone knows that they can take over any assigned issue, usually as long as it is not in progress.

Maybe it's better to simply view the component-owner as the 'resident-expert' in each area, and probably  the person with the most knowledge on the subject. I think we should leave auto-assign as is, with the exception of maybe @stevengill could drop some of his plugins.

**Comment by stevengill** [comment]
+1 to removing plugin auto assigning.

**Comment by shazron** [comment]
LGTM

**Comment by csantanapr** [comment]
+1 LGTM 
Yes Agree, I think folks see a name associated with a bug, and they think that person already looked at it when it reality it's what @purplecabbage indicated it's all for grabs, and the active folks are more resident experts.

I think it's great doc @nikhilkh and when we agree it can be put in the "Contribute" page on the website at least the portion of how to open a bug and what to do with the fields.

**Pull request closed without merging**